app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimAmountPage.scala.html

```diff
@@ -55,21 +56,25 @@
   @formHelper(action = controllers.routes.SubsidyController.postAddClaimAmount()) {
 
     @H1(messages("add-claim-amount.title"))
-    @P(messages("add-claim-amount.p1"))
-    @P(messages("add-claim-amount.p2"))
+    @P(messages("add-claim-amount.p1", "https://ec.europa.eu/info/funding-tenders/procedures-guidelines-tenders/information-contractors-and-beneficiaries/exchange-rate-inforeuro_en"))
```

Comment:
Should we consider externalising this url?

Reply:
it was like this so I got lazy and copy pasted  🤣 . I can do that

Reply:
updated [here](https://github.com/hmrc/eu-subsidy-compliance-frontend/pull/78/commits/7c6d711ea23139e0d75dac92ebcb27d02364c37d)